test/test_integration_subdoc.cxx

```diff
@@ -619,15 +621,6 @@ TEST_CASE("integration: subdoc multi mutation", "[integration]")
         assert_single_lookup_success(integration, couchbase::protocol::subdoc_opcode::get, id, "counter", "42");
     }
 
-    SECTION("mismatched type and opcode")
-    {
-        couchbase::operations::mutate_in_request req{ id };
-        req.specs.add_spec(couchbase::protocol::subdoc_opcode::get, false, "p");
-        req.specs.add_spec(couchbase::protocol::subdoc_opcode::get, false, "p");
-        auto resp = test::utils::execute(integration.cluster, req);
-        REQUIRE(resp.ctx.ec == couchbase::error::common_errc::invalid_argument);
-    }
-
```

Comment:
@jrawsthorne I removed this test, because it is considered programmer error, and contract violation. The library will generate assertion and terminate program. Our test framework does not allow to verify death conditions.